qemu/tests/cfg/qemu_img_negative.cfg

```diff
@@ -20,8 +20,8 @@
                     host_setup_flag = 2
                     #Please update portal_ip, initiator and target params based on your test env
                     #portal_ip = 127.0.0.1
-                    #initiator = "iqn.2010-07.com.redhat:kvmautotest"
-                    #target = "iqn.2001-05.com.equallogic:0-8a0906-db31f7d03-470263b05654c204-kvm-autotest"
+                    #initiator = "iqn.2010-07.com.example:my_initiator"
+                    #target = "iqn.2001-05.com.mytarget:0-0a0000-000000000-0000000000000000-my-target"
```

Comment:
Yep, good. Only please remove the `#` as without these the test simply uses the default local disk(s)...